framework/include/timeintegrators/TimeIntegrator.h

```diff
@@ -45,6 +45,24 @@ class TimeIntegrator : public MooseObject, public Restartable
 public:
   TimeIntegrator(const InputParameters & parameters);
 
+  /**
+   * Called to setup datastructures.
+   *
+   * WILL be called during recover/restart
+   *
+   * Should NOT do any computation for initial values
+   * use init() for that
+   *
+   * Note: this doesn't inherit this from SetupInterface because
```

Comment:
"...this does not inherit from..." (extra this)

Reply:
I see. How about "This method is not inherited from SetupInterface,..."?

Reply:
It's fine